third_party/cares/cares.BUILD

```diff
@@ -106,6 +111,7 @@ copy_file(
         ":ios_armv7": "@com_github_grpc_grpc//third_party/cares:config_darwin/ares_config.h",
         ":ios_armv7s": "@com_github_grpc_grpc//third_party/cares:config_darwin/ares_config.h",
         ":ios_arm64": "@com_github_grpc_grpc//third_party/cares:config_darwin/ares_config.h",
```

Comment:
just curious - what's the difference between ios_sim_arm64 and ios_arm64? can the latter be used?

Reply:
I believe ios_sim_arm64 is for simulator on M1 and ios_arm64 is for device.
I tried running with ios_arm64 but failed with `incompatible platform (have 'iOS', need 'iOS-sim')`

Reply:
Ah thanks for explaining :)